### ISSUE TITLE
run on Docker,seqclean (all_transcripts.fasta) encountered an error about Error at 'psx -p ...

### ISSUE THREAD

**Issue by wangnan9394** [PASApipeline/PASApipeline#155]
when use docker for command "seqclean",there is an error,same as link:[http://seqanswers.com/forums/showthread.php?t=8324](http://seqanswers.com/forums/showthread.php?t=8324)

ERROR:
```
seqclean running options: 
seqclean all_transcripts.fasta
 Standard log file: seqcl_all_transcripts.fasta.log
 Error log file:    err_seqcl_all_transcripts.fasta.log
 Using 1 CPUs for cleaning
-= Rebuilding all_transcripts.fasta cdb index =-
 Launching actual cleaning process:
 psx -p 1  -n 1000  -i all_transcripts.fasta -d cleaning -C '/usr/local/src/PASApipeline/sample_data/all_transcripts.fasta:ANLMS100:::11:0' -c '/usr/local/src/PASApipeline/bin/seqclean.psx'
Error at 'psx -p 1  -n 1000  -i all_transcripts.fasta -d cleaning -C '/usr/local/src/PASApipeline/sample_data/all_transcripts.fasta:ANLMS100:::11:0' -c '/usr/local/src/PASApipeline/bin/seqclean.psx''


Process terminated with an error!
seqclean (all_transcripts.fasta) encountered an error.
Working directory was /usr/local/src/PASApipeline/sample_data
seqclean (all_transcripts.fasta) encountered an error.

```
when into the container, and test in sample_data, my command line:
`root@cc7419c1838f:/usr/local/src/PASApipeline/sample_data# ../bin/seqclean all_transcripts.fasta`

I think it must be about the hard link, but i don't know how to fix it.

**Comment by wangnan9394** [comment]
link to ERROR#73 https://github.com/PASApipeline/PASApipeline/issues/73 in docker and link to https://groups.google.com/g/trinityrnaseq-users/c/0wAt0fUQo2E in general. ; )

**Comment by charles-plessy** [comment]
I had the same error when running the fixed `seqclean`from PASA `2.4.1` in _Nextflow_. It turned out that _Nextflow_ clears most of the environment variables before running the command, but `seqclean` requires `$USER` to be set or crashes like above…

**Comment by colindaven** [comment]
I can second the comment from @charles-plessy about the user when using Nextflow.

Here is my workaround when using the PASA docker container - change `YOUR_LINUX_USER` as appropriate

```
    """
    export PASAHOME=/usr/local/src/PASApipeline
    export PATH=/usr/local/src/PASApipeline:$PATH
    export USER=YOUR_LINUX_USER
    
    seqclean $transcripts -c $task.cpus
    """
```
